### PR TITLE
Move the SDK Performance tests to Helix

### PR DESCRIPTION
**Pull request by ghost**
Move the SDK Performance tests to Helix

@MattGal
@DrewScoggins
@adiaaida

### CODE REVIEW

**Comment by johnbeisner** [comment]
Reference: https://dnceng.visualstudio.com/internal/_build/results?buildId=165297

File: eng/build.yml [hunk 1]
```diff
@@ -88,26 +88,59 @@ jobs:
...
+          displayName: Selection of sources
+        - template: /eng/common/templates/steps/send-to-helix.yml
+          parameters:
+            HelixSource: 'pr/dotnet/sdk/$(Build.SourceBranch)'
+            HelixType: 'test/sdk_$(_BuildConfig)/'
+            HelixAccessToken: $(HelixApiAccessToken)
+            HelixTargetQueues: 'Windows.10.Amd64.ClientRS1.Perf' # using a dedicated private Helix queue (perfsnakes)
+            HelixPreCommands: 'set "BV_UPLOAD_SAS_TOKEN=$(BenchViewUploadToken)"'
+
+            WorkItemTimeout: 4:00
```
Comment (adiaaida) [inline eng/build.yml]: How long does your perf testing run? Does this need to be 4 hours? Can it be shorter?

Comment (johnbeisner) [reply eng/build.yml]: About 1 hour 15 minutes - I'll shorten it up.

**Comment by johnbeisner** [comment]
Reference: https://mc.dot.net/#/user/dotnet-mc-bot-2/pr~2Fdotnet~2Fsdk~2Frefs~2Fheads~2Fperftestsonhelix/test~2Fsdk_release~2F/20190423.3

**Comment by adiaaida** [comment]
Just for an FYI: it looks like your test did upload to benchview. I am seeing this in the search page: SDK perf Ubuntu_16.04 x64 Release rolling refs/heads/PerfTestsOnHelix 827232443b5fad714ef662c54e39f54f249f8c71 -- this looks like your branch.

**Pull request merged**